scripts/migrations/055_deactivateEmailNotification.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2023. Part of the SW360 Portal Project.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# This is a manual database migration script. It is assumed that a
+# dedicated framework for automatic migration will be written in the
+# future. When that happens, this script should be refactored to conform
+# to the framework's prerequisites to be run by the framework. For
+# example, server address and db name should be parameterized, the code
+# reorganized into a single class or function, etc.
+#
+# This script is for deactivating email notification for user not belonging to a domain.
+# ---------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+import time
+import datetime
+import couchdb
+import json
+from webbrowser import get
+
+# ---------------------------------------
+# constants
+# ---------------------------------------
+
+DRY_RUN = True
+
+COUCHSERVER = 'http://username:password@localhost:5984/'
+USERS_DB = 'sw360users'
+
+couch = couchdb.Server(COUCHSERVER)
+userDb = couch[USERS_DB]
+
+#Users
+NOTIFICATION = 'wantsMailNotification'
+EMAIL = 'email'
+
+####################################################################################################
+#Add the Domains that should belong to a particular instance and seperate them by using '|' operator
+DOMAINS = "@siemens.com|@siemens-energy.com|@siemens-healthineers.com|@gmail.com|sw360.org"
```

Comment:
Please remove the siemens reference domain names may be only "@<domain>.com|@<domain>.org",  
also the comment can be more explicit like:
"#all email from the domains not included in list (separated with '|' ) will be disabled for email notification"

Reply:
done the changes.